lualib/resty/events/frame.lua

```diff
@@ -82,12 +89,17 @@ function _M.send(sock, payload)
         return nil, err
     end
 
-    local bytes, err = sock:send(uint16_to_bytes(payload_len) .. payload)
+    local data = SEND_DATA
+    data[1] = uint_to_bytes(payload_len)
+    data[2] = payload
+
+    local bytes, err = sock:send(data)
```

Comment:
For the current implementation of `lua-nginx-module`, I think there is no race condition on `SEND_DATA`. 

Just mention that changing data during the call of `sock:send()` is undefined behavior, but I think it is not a blocker of this PR.